app/config/question-bank.js

```diff
@@ -144,6 +144,12 @@ const questionBank = {
           url: 'applicant',
           baseUrl: 'applicant',
           preValidationKeys: ['projectSubject'],
+          // preValidationObject: {
+          //    preValidationKey: ['projectSubject'],
+          //    preValidationAnswer: ['project-subject-A1'],
+          //    preValidationRule: 'AND',
+          //    preValidationUrls: ['project-subject']
+          // }
```

Comment:
can we remove these comments please @zavies97

Reply:
not yet as this is what will be added to question bank, and i havent actually added the pre validation yet as cant test it